src/domain/validator.js

```diff
@@ -25,6 +24,13 @@ export const validateBonusNumer = (winningNumbers, bonusNumber) => {
   validateBonusDuplicate(winningNumbers, bonusNumber);
 };
 
+export const validateRetry = (retry) => {
+  retry = retry.toLowerCase();
```

Comment:
```suggestion
export const validateRetry = (retry) => {
  const lowerRetryKeyword = retry.toLowerCase();
```

파라미터 값을 임의 조작하는 행위는 꼭 지양하면 좋겠습니다!